easy-ui-react/src/Button/Button.module.scss

```diff
@@ -1,5 +1,5 @@
-@use "./mixins" as Button;
-@import "../styles/common";
+@use "../styles/common" as *;
```

Comment:
Swapped the order because it felt better to have it come before the namespaced `@use`

Reply:
agreed!